packages/build/package.json

```diff
@@ -23,7 +23,12 @@
   "scripts": {
     "prepublishOnly": "cd ../../ && npm run prepublishOnly",
     "prebuild": "rm -rf lib",
-    "build": "cp -a src lib/"
+    "build": "cp -a src lib/",
+    "test:dev": "run-s test:dev:*",
```

Comment:
I think this can be simplyfied. Would love to get rid of the `run-s` and `run-p`. For chaining things we should now use lerna